### PR TITLE
Bump Version 2.0.1

### PR DESCRIPTION
**Pull request by jasoet-gdp**
Use Javaslang 2.0.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Developer versions can be found [here](https://oss.sonatype.org/content/reposito
 <dependency>
   <groupId>io.javaslang</groupId>
   <artifactId>javaslang-jackson</artifactId>
-  <version>2.1.0-SNAPSHOT</version>
+  <version>2.0.1</version>
```
Comment (danieldietrich) [inline README.md]: The developer version still remains 2.1.0-SNAPSHOT

File: README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ Ensure that your `~/.m2/settings.xml` contains the following:
 ### Gradle
 
 ```groovy
-compile("io.javaslang:javaslang-jackson:2.1.0-SNAPSHOT")
+compile("io.javaslang:javaslang-jackson:2.0.1")
```
Comment (danieldietrich) [inline README.md]: Dito

File: pom.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
     <groupId>io.javaslang</groupId>
     <artifactId>javaslang-jackson</artifactId>
-    <version>2.1.0-SNAPSHOT</version>
+    <version>2.0.1</version>
```
Comment (danieldietrich) [inline pom.xml]: The developer version still remains 2.1.0-SNAPSHOT

File: pom.xml [hunk 4]
```diff
@@ -40,7 +40,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
         <junit.version>4.12</junit.version>
-        <javaslang.version>2.0.0</javaslang.version>
+        <javaslang.version>2.0.1</javaslang.version>
```
Comment (danieldietrich) [inline pom.xml]: but this is OK!

**Comment by jasoet-gdp** [comment]
@danieldietrich OK i get it now :D

**Comment by danieldietrich** [comment]
Many thanks for updating the versions, that helps!

**Comment by ruslansennov** [comment]
@danieldietrich I have credentials only on my home computer, so please could you deploy jackson module to oss repository?

**Comment by danieldietrich** [comment]
yep, will do!

**Comment by danieldietrich** [comment]
oh, merged the PR - maybe you would have waited until push to maven central. however, will be there neartime...

**Comment by danieldietrich** [comment]
it is released, will appear in maven central soon...

**Pull request merged**